pipeline/backend/types/step.go

```diff
@@ -48,6 +48,7 @@ type Step struct {
 	NetworkMode    string            `json:"network_mode,omitempty"`
 	IpcMode        string            `json:"ipc_mode,omitempty"`
 	Sysctls        map[string]string `json:"sysctls,omitempty"`
+	Ports          []int32           `json:"ports,omitempty"`
```

Comment:
```suggestion
	Ports          []uint16           `json:"ports,omitempty"`
```

ports are uint16 if we want to be specific :)